app/controllers/application_controller.rb

```diff
@@ -169,11 +169,11 @@ def instrument_with_timer
   # given cache_key and fills out custom response headers.
   # If the attempt results in a cache miss, then the response is generated normally and
   # will be stored in the cache.
-  def fetch_from_or_store_in_cache(skip_cache, cache_key, httpdate, event_name)
+  def fetch_from_or_store_in_cache(skip_cache, cache_key, httpdate)
```

Comment:
Wait, why did we get rid of the event name? 
We want this to be a generic function for loading from cache. So, we should still use an even t name.

Reply:
Got it, added it back in.

One thing to note is that the old report page used `samples` as the event name, whereas the refactored version previously used `PipelineReport`. I switched it back over to `samples` in this last commit for the sake of consistency, but if there's a strong preference for moving to `PipelineReport` as the new event name then I'm happy to do that!

Reply:
Let's choose what makes more sense long term. 
My opinion is that samples is too generic... that could be a problem in the future if we choose to cache other stuff about samples.	 And that the event should indicate that it is the report info that is being cached. Would also be good to have a standard way of naming these. Maybe the controller + action name? For instance, `samples_report` (we should also respect Datadog's naming constraints: https://docs.datadoghq.com/developers/metrics/#naming-custom-metrics).
Wdyt?

Reply:
Sounds good to me! I looked through a few of the other event names we have and it seems they typically use . instead of _ to separate controller + action, so I'll be updating to `samples.report`.

Reply:
They have different meaning in Datadog. They actually a sort of hierarchy for the metrics keys.
Having said that, I agree, seems to make sense to have `controller.action`.